src/ctia/entity/tool.clj

```diff
@@ -1,5 +1,6 @@
 (ns ctia.entity.tool
   (:require [ctia.entity.tool.schemas :as ts]
+            [ctia.entity.tool.graphql-schemas :as tgs]
```

Comment:
That ns is not used

Reply:
unfortunately it is implicitly used, loading the ns loads the GraphQL type, otherwise you get the default one from CTIM which doesn't include the custom fields.